peachjam/admin.py

```diff
@@ -1121,11 +1121,28 @@ def get_formset(self, request, obj=None, **kwargs):
 
 class JudgmentAdminForm(DocumentForm):
     hearing_date = forms.DateField(widget=DateSelectorWidget(), required=False)
+    held = forms.CharField(widget=CKEditorWidget(), required=False)
```

Comment:
These are rich HTML fields, they're just plain text. So no need for CKEditor widget.